### PR TITLE
TSPS-326 add wdl to mask or subset a vcf given a bed file

### PR DESCRIPTION
**Pull request by jsotobroad**
### Description 

We want to use this wdl for scientific validation of the refernece panel.  We will mask wgs samples down to an array build's sites and then impute and we shoudl get something close to the original wgs sample

Example of workflow running - https://bvdp-saturn-dev.appspot.com/#workspaces/general-dev-billing-account/tsps_gcp_scratch_space_mma/job_history/dd9e56bd-63d6-43d4-bd9f-93caf91dcdae

### Jira Ticket
https://broadworkbench.atlassian.net/browse/TSPS-326

### CODE REVIEW

File: pipelines/imputation/scientificValidation/README.md [hunk 1]
```diff
@@ -13,3 +13,18 @@ This wdl is basically a wrapper around that tool/image
 #### Outputs
 * recombined_reference_panel - output vcf after mitigation 
 algorithm has been run
+
+
+## ReshapeReferencePanel
```
Comment (mmorgantaylor) [inline pipelines/imputation/scientificValidation/README.md:18]:
```suggestion
## SubsetVcfByBedFile
```

**Review by mmorgantaylor — COMMENTED** [review]
Is the only thing we care about that the proper sites are filtered down? do we care about headers or annotations or anything like that that are expected for an array but wouldn't be present in a WGS vcf?

have you run the output of this through imputation?

**Comment by jsotobroad** [comment]
> Is the only thing we care about that the proper sites are filtered down? do we care about headers or annotations or anything like that that are expected for an array but wouldn't be present in a WGS vcf?
> 
> have you run the output of this through imputation?

Outside of the dictionary lines im not sure if there are any other significant headers that we should care about

**Review by mmorgantaylor — APPROVED** [review]
please add a couple lines in the readme to note that we don't care about headers/annotations because the imputation tools don't care about those either.

**Pull request merged**